django-cloudlaunch/cloudlaunch/backend_plugins/base_vm_app.py

```diff
@@ -227,8 +227,14 @@ def _setup_networking(self, provider, net_id, subnet_id, placement):
                 provider, net_id, placement)
         # Make sure the subnet has Internet connectivity
         try:
-            found_routers = [router for router in provider.networking.routers
-                             if router.network_id == subnet.network_id]
+            # This will allow to re-use a router when subnets from multiple networks are attached to it
```

Comment:
```suggestion
            # This will allow to re-use a router when subnets from multiple networks are attached to it
            # xref: https://github.com/galaxyproject/cloudlaunch/pull/261
```